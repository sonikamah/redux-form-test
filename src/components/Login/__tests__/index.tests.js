import React from 'react';
import Login from '../index';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import update from 'react-addons-update';

test('Login Component snapshot', () => {
    const login = renderer.create(
        <Login/>
    );
    let tree = login.toJSON();
    expect(tree).toMatchSnapshot();

});

