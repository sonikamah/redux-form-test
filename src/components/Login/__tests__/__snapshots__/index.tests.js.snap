exports[`test Login Component snapshot 1`] = `
<div
  className="login-container">
  <div
    className="loginRow-spacing row">
    <div
      className="col-md-offset-8 col-md-3">
      <form
        onSubmit={[Function]}>
        <div
          className="form-login">
          <h4>
            User Login
          </h4>
          <input
            className="form-control input-sm chat-input"
            name="username"
            onChange={[Function]}
            placeholder="username"
            type="text"
            value="" />
          <p
            className="error-message">
            
          </p>
          <br />
          <input
            className="form-control input-sm chat-input"
            name="password"
            onChange={[Function]}
            placeholder="password"
            type="password"
            value="" />
          <p
            className="error-message">
            
          </p>
          <br />
          <div
            className="wrapper">
            <span
              className="group-btn">
              <button
                className="btn btn-lg btn-primary"
                type="submit"
                value="Submit">
                Sign In
              </button>
               
              <i
                className="fa fa-sign-in" />
            </span>
          </div>
        </div>
      </form>
    </div>
  </div>
</div>
`;
